js/remove_string_spaces.test.js

```diff
@@ -0,0 +1,13 @@
+const noSpaces = require('./remove_string_spaces');
+
+
+describe('remove_string_spaces', () => {
+
+  test('1 case', () => {
```

Comment:
How about naming the test cases in a more specific manner? 

For example, this one could be called _with single spaces_ and the second one could be called _with multiple spaces_